src/libcore/mem.rs

```diff
@@ -1056,12 +1056,22 @@ impl<T: ?Sized> DerefMut for ManuallyDrop<T> {
 /// This is exploited by the compiler for various optimizations, such as eliding
 /// run-time checks and optimizing `enum` layout.
```

Comment:
Note to self (or to anyone else): The first line in the type docs says: "A newtype...".
This type is a wrapper, but definitely not a newtype, so it should say "A wrapper for constructing..."